llvm/unittests/Target/AMDGPU/MachineInstrBundleTest.cpp

```diff
@@ -0,0 +1,61 @@
+//===- MachineInstrBundleTest.cpp -----------------------------------------===//
```

Comment:
See `test/CodeGen/AMDGPU/finalizebundle.mir` for a much nicer lit-based way to test finalizeBundle.

Reply:
Indeed, that is a lot nicer